nntrainer/tensor/tensor.cpp

```diff
@@ -745,32 +745,59 @@ Tensor Tensor::getSharedDataTensor(const TensorDim dim_, unsigned int offset,
   return ret;
 }
 
-std::vector<Tensor> Tensor::split(unsigned num_size, int axis) {
+std::vector<Tensor> Tensor::split(std::vector<unsigned int> sizes, int axis) {
```

Comment:
how about using offset? and make an additional function that uses this?
``` c++
std::vector<Tensor> Tensor::split(std::vector<size_t> offset, int axis){...};

// This is to split tensor evenly.
std::vector<Tensor> Tensor::split(unsigned int num_size, int axix){
    std::vector< size_t> offsets.emplace_back(num_size) ;
   offsets[0] = 0;
   size_t size = dim.getTensorDim(axis) /num_size;
   for(unsigned int j = 1; j<num_size ;j++)
        offsets[j] = offsets[i] + size;
   
   return split( offset,  axis); 
}


```

Reply:
Added additional function as you recommend.
Split can be done by either way of using offset, sizes. (numpy use offset, tensorflow, pytorch use size).
If there is specific reason to using offset please let me know. I will change that way.

Reply:
If other frameworks use the size, then we could go with it then.